usaspending_api/search/filters/elasticsearch/tas.py

```diff
@@ -11,7 +11,7 @@ class TasCodes(_Filter, HierarchicalFilter):
     underscore_name = "tas_codes"
 
     @classmethod
-    def generate_elasticsearch_query(cls, filter_values, query_type: _QueryType, nested_path: str = "") -> ES_Q:
+    def generate_elasticsearch_query(cls, filter_values, query_type: _QueryType, **options) -> ES_Q:
         if isinstance(filter_values, list):
             # This is a legacy usage, and will be dealt with by the other filter
             return TreasuryAccounts.generate_elasticsearch_query(filter_values, query_type)
```

Comment:
Why not give the `TreasuryAccounts.generate_elasticsearch_query(filter_values, query_type)` method an `options` parameter?

Reply:
Simply overlooked it when updating all methods that implemented `_Filter`. Will get that changed.